bin/thin_client_jaas.conf

```diff
@@ -0,0 +1,5 @@
+ThinClient {
```

Comment:
Maybe make this PhoenixThinClient? Disambiguate from other products who have JAAS config names.

e.g. ZooKeeper uses a generic name like "Client" or something equally annoying and likely to conflict with other systems :)

Reply:
I did, but since it's internal only, I don't think it matters much.

Reply:
I mean I've moved this to java code.